frontend/src/App.jsx

```diff
@@ -0,0 +1,47 @@
+import { Route, Routes, Navigate } from "@solidjs/router";
```

Comment:
We have to redirect from `/3d-scene-editing-app` to `/3d-scene-editing-app/train` instead of `/train`